src/services/routes/index.js

```diff
@@ -7,21 +7,31 @@ router.get("/", (req, res, next) => {
   res.sendFile(path.join(__dirname, "..", "..", "dist", "index.html"));
 });
 
-router.get("/api/v1/create/routes", (req, res, next) => {
+router.get("/api/v1/hello", (req, res, next) => {
+  res.send({ message: "Hello world!" });
+});
+
+router.get("/api/v1/routes", (req, res, next) => {
+  // NOTE: This will not work unless the user has been created
+  // Add in better error handling!
   getData("SELECT * FROM routes;").then(results => {
     if (results.error) {
+      console.log(results.error);
       res.status(404).send({ error: results.error });
     } else {
+      console.log(results.data);
       res.send({ body: results.data });
     }
   });
 });
 
-router.post("/api/v1/create", (req, res, next) => {
+router.post("/api/v1/user", (req, res, next) => {
+  // NOTE: This will not work unless the user has been created
```

Comment:
Delete this comment!

Reply:
(Also this route is to create the user, so the comment definitely doesn't apply) ;)